app/herms.hs

```diff
@@ -197,6 +197,13 @@ getServingsAndConv serv convName config = (servings, conv)
           | convName  == t Str.imperial =  Imperial
           | otherwise = defaultUnit' config
 
+findRecipes :: String -> HermsReader IO ()
+findRecipes rgx = do -- TODO: add colour option here, maybe modify findMatches to just return a list of tuples where we can colorify the first part?
```

Comment:
This line is very long :smile: Let's go ahead and track colorization on https://github.com/LuxMiranda/herms/issues/36 or a follow-up, where we can fit more details, use labels, etc.

Reply:
Sorry about that! Autowrapping is a major hindrance sometimes